<launch>

 <!-- <include file="$(find ghattas_navigation)/launch/rs_t265.launch"/> -->
 <node name="map_server" pkg="map_server" type="map_server" args="$(find ghattas_navigation)/map/empty.yaml"/>
 <rosparam command="load" file="$(find ghattas_navigation)/param/navigation.yaml" />
 <include file="$(find ghattas_navigation)/launch/move_base_launcher.launch"/>
 <node pkg="ghattas_navigation" type="odometry_publisher.py" name="odometry_publisher" output= "screen"/>
 <node pkg="ghattas_navigation" type="twist_to_thrusters.py"  name="twist_to_thrusters" output="screen"/>
 <node pkg="ghattas_navigation" type="navigation_node.py"  name="navigation_node" output="screen"/>
 <node type="rviz" name="rviz" pkg="rviz" args="-d $(find ghattas_navigation)/rviz/navigation.rviz" />

</launch>
